sei/chain.json

```diff
@@ -0,0 +1,118 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "sei",
+    "status": "live",
+    "website": "https://www.sei.io/",
+    "network_type": "mainnet",
+    "pretty_name": "Sei",
+    "chain_id": "pacific-1",
+    "bech32_prefix": "sei",
+    "daemon_name": "seid",
+    "node_home": "$HOME/.sei",
+    "key_algos": [
+        "secp256k1"
+    ],
+    "slip44": 118,
+    "fees": {
+        "fee_tokens": [
+            {
+                "denom": "usei",
+                "fixed_min_gas_price": 0,
+                "low_gas_price": 0,
+                "average_gas_price": 0.025,
+                "high_gas_price": 0.04
+            }
+        ]
+    },
+    "staking": {
+        "staking_tokens": [
+            {
+                "denom": "usei"
+            }
+        ]
+    },
+    "codebase": {
+        "git_repo": "https://github.com/sei-protocol/sei-chain",
+        "recommended_version": "v3.0.6",
+        "compatible_versions": [
+            "v3.0.6"
+        ],
+        "cosmos_sdk_version": "",
+        "cosmwasm_enabled": true,
+        "cosmwasm_path": "$HOME/.sei/wasm",
+        "genesis": {
+            "genesis_url": "https://snapshots.lavenderfive.com/genesis/sei/genesis.json"
+        },
+        "versions": [
+            {
+                "name": "",
+                "recommended_version": "v3.0.6",
+                "compatible_versions": [
+                    "v3.0.6"
+                ],
+                "cosmos_sdk_version": "",
+                "cosmwasm_enabled": false,
+                "cosmwasm_path": "$HOME/.sei/wasm"
+            }
+        ]
+    },
+    "logo_URIs": {
+        "png": "",
+        "svg": ""
+    },
+    "peers": {
+        "seeds": [
+            {
+                "id": "20e1000e88125698264454a884812746c2eb4807",
+                "address": "seeds.lavenderfive.com:11956",
+                "provider": "Lavender.Five Nodes 🐝"
+            }
+        ],
+        "persistent_peers": [
+            {
+                "id": "20e1000e88125698264454a884812746c2eb4807",
```

Comment:
what's the recommended practice for this, is the general best practice to have active set vals here? or just a handful of peers?

Reply:
I actually recommend zero peers, as seeds are far superior, but inevitably people will add them, so I throw one in there to start the process.